news/2021-06-14-mca-ayim-2019.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Mapper's Choice Awards and A Year In Mapping 2019"
+date: 2021-06-14 00:00:00 +0000
+---
+
+Map connoisseurs rejoice, because the *Mappers' Choice Awards* AND *A Year In Mapping* (AYIM) are back and again and rearing to catch up with the scene!
```

Comment:
```suggestion
Map connoisseurs rejoice, because the *Mappers' Choice Awards* (MCA) AND *A Year In Mapping* (AYIM) are back again and raring to catch up with the scene!
```